src/main/java/net/datafaker/idnumbers/IdNumbers.java

```diff
@@ -1,7 +1,24 @@
 package net.datafaker.idnumbers;
 
+import net.datafaker.providers.base.BaseProviders;
+
 import java.time.format.DateTimeFormatter;
 
 public interface IdNumbers {
     DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyMMdd");
+
+    /**
+     * @return ISO-2 code of the country ("US" for America, "EE" for Estonia etc.)
+     */
+    String country();
```

Comment:
would countryCode be perhaps a better named method? A method like `country()` might return "America" or so.

Reply:
Also, shouldn't this class perhaps be called "IdNumber", instead of "IdNumbers" (so, without the `s`?)

Reply:
@bodiam I like both variants: `countryCode` and `IdNmberGenerator`. Will rename.

Reply:
Nice, thanks!